lib/ContextualPopup/ContextualPopup.js

```diff
@@ -533,7 +549,7 @@ module.exports = kind(
 	* @private
 	*/
 	showHideScrim: function (inShow) {
-		if (this.floating && (this.scrim || (this.modal && this.scrimWhenModal))) {
+		if (this.floating && (this.scrim || this.modal || this.scrimWhenModal)) {
```

Comment:
When the model is true then the if condition will be true and scrim.showAtZIndex() will execute. But when model turns to false the if condition will return false and the scrim.hideAtZIndex() won't execute. the model and scrimWhenModal again verified again inside. ideally the model shouldn't check in this if condition here.